.github/workflows/plugin-release.yml

```diff
@@ -0,0 +1,51 @@
+name: Plugin-Release
+
+on: pull_request
+
+jobs:
+
+  ubuntu:
+    runs-on: ubuntu-18.04
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Replace token 1
+      run: sed -i -e "s,placeholder_rudder_dataplane_url,${{ secrets.RUDDER_DATAPLANE_URL }},g" ${{ github.workspace }}/server/services/telemetry/telemetry.go
+
+    - name: Replace token 2
+      run: sed -i -e "s,placeholder_rudder_key,${{ secrets.RUDDER_PROD_KEY }},g" ${{ github.workspace }}/server/services/telemetry/telemetry.go
+
+    - name: npm install
+      run: cd webapp; npm install --no-optional
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15
+
+    - name: Set up Node
+      uses: actions/setup-node@v2
+      with:
+        node-version: 16.1.0
+
+    - name: Build webapp
+      run: make webapp
+
+    - name: npm install plugin dependencies
+      run: cd mattermost-plugin/webapp; npm install --no-optional
+
+    - name: Build plugin
+      run: cd mattermost-plugin; make dist
+      env:
+        BUILD_NUMBER: ${{ github.run_id }}
+
+    - name: Rename plugin file
+      run: cd mattermost-plugin/dist; mv focalboard-0.6.7.tar.gz mattermost-plugin-focalboard-0.6.7.tar.gz
```

Comment:
This probably should not have the version number hardcoded in the file name. Is the version necessary for the plugin infrastructure? If so, that's ok for now. Longer-term, we will need to consolidate our version numbers anyway.

Reply:
I was hesitant to add it at first, but all the plugins are released with their version numbers. What we can do is to abstract this job to generate an artifact without the version number, but my understanding is that we would have to rename the artifact before uploading it.

I'll make a quick test on how the no-version-number approach would look like, and we can iterate over it

Reply:
Addressed in c58156c